### PR TITLE
remove redundant translations

### PR DESCRIPTION
**Pull request by m4ra**

### CODE REVIEW

File: apps/userdashboard/assets/js/a4_candy_userdashboard/UserReport.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ export const UserReport = (props) => {
     return (
       <p>
         {splitted[0]}
-        <b>{splitted[1]}</b>
+        <b>{userName}</b>
```
Comment (m4ra) [inline apps/userdashboard/assets/js/a4_candy_userdashboard/UserReport.jsx:26]: I dunno why this got altered by the py linter. @goapunk anyhow it is correct.

Comment (goapunk) [reply apps/userdashboard/assets/js/a4_candy_userdashboard/UserReport.jsx:26]: ups, nice catch!

**Review by goapunk — CHANGES_REQUESTED** [review]
@m4ra did you accidentally amend the changes instead of making a new commit? because it's not all in the last commit on main (hence the weird username changes). Could you move the changes into a new commit ?

**Comment by goapunk** [comment]
@m4ra @philli-m should we also delete the locale folders of the unused languages in `locale-source` and locale-fork` as they are still updated when running `make po` otherwise

**Comment by philli-m** [comment]
@goapunk probably would be good although would it make rebase more annoying?

**Comment by m4ra** [comment]
`@m4ra did you accidentally amend the changes instead of making a new commit? because it's now all in the last commit on main (hence the weird username changes). Could you move the changes into a new commit ?`
uh yeah i did, I will update the commits. and delete the loales too. Why would that make rebase annoying?

**Comment by goapunk** [comment]
> > @m4ra did you accidentally amend the changes instead of making a new commit? because it's now all in the last commit on main (hence the weird username changes). Could you move the changes into a new commit ?
> 
> uh yeah i did, I will update the commits. and delete the loales too. Why would that make rebase annoying? @goapunk @philli-m

i guess because there will always be the conflict between changes and deletion, but it is easy to resolve as we want to keep the deletion anyway

**Review by goapunk — APPROVED** [review]
nice!

**Pull request merged**